### PR TITLE
status.get descriptions

### PR DESCRIPTION
**Pull request by dmitrydao**
## Overview

Changes to branding and `status.get` description.

## Changes

- Added descriptions to `status.get` response body.
- Updated email and company name across all versions.

### CODE REVIEW

File: docs-ui/1.0.0/security_critical_api_specs.yaml [hunk 1]
```diff
@@ -345,27 +345,34 @@ paths:
                           last_validated_child_block_timestamp:
                             type: integer
                             format: int64
+                            description: The time of the last Watcher sync with the child chain.
                           last_validated_child_block_number:
                             type: integer
                             format: int64
+                            description: The last block that the Watcher has synced with the child chain.
```
Comment (okalouti) [inline docs-ui/1.0.0/security_critical_api_specs.yaml:352]: Perhaps specify _child chain_ block in the descriptions for the child chain ones?

Comment (dmitrydao) [reply docs-ui/1.0.0/security_critical_api_specs.yaml:352]: done

File: docs-ui/1.0.0/security_critical_api_specs.yaml [hunk 2]
```diff
@@ -345,27 +345,34 @@ paths:
...
                             format: int64
+                            description: The last child chain block that has been seen on smart contracts.
                           last_seen_eth_block_timestamp:
                             type: integer
                             format: int64
+                            description: The time of the last Ethereum block that has been seen by the Watcher.
                           last_seen_eth_block_number:
                             type: integer
                             format: int64
+                            description: The last Ethereum block that has been by the Watcher.
```
Comment (unnawut) [inline docs-ui/1.0.0/security_critical_api_specs.yaml:368]: been by -> been seen by

File: docs-ui/master/security_critical_api_specs.yaml [hunk 3]
```diff
@@ -345,27 +345,34 @@ paths:
...
                             format: int64
+                            description: The last child chain block that has been seen on smart contracts.
                           last_seen_eth_block_timestamp:
                             type: integer
                             format: int64
+                            description: The time of the last Ethereum block that has been seen by the Watcher.
                           last_seen_eth_block_number:
                             type: integer
                             format: int64
+                            description: The last Ethereum block that has been by the Watcher.
```
Comment (unnawut) [inline docs-ui/master/security_critical_api_specs.yaml:368]: been by -> been seen by

**Comment by unnawut** [comment]
Also one last favour. Could you also reflect these changes onto https://github.com/omgnetwork/elixir-omg/tree/master/apps/omg_watcher_rpc/priv/swagger?

This branch has only the compiled version and so all these changes would get overridden by future doc re-compilations.

**Comment by dmitrydao** [comment]
> Also one last favour. Could you also reflect these changes onto https://github.com/omgnetwork/elixir-omg/tree/master/apps/omg_watcher_rpc/priv/swagger?
> 
> This branch has only the compiled version and so all these changes would get overridden by future doc re-compilations.

Ok. Do I need to change only these two files?
- https://github.com/omgnetwork/elixir-omg/blob/master/apps/omg_watcher_rpc/priv/swagger/info_api_specs.yaml
- https://github.com/omgnetwork/elixir-omg/blob/master/apps/omg_watcher_rpc/priv/swagger/security_critical_api_specs.yaml

**Comment by unnawut** [comment]
Ah sorry, these are the files you need to update:

Emails, company names and urls:
- https://github.com/omgnetwork/elixir-omg/blob/master/apps/omg_watcher_rpc/priv/swagger/info_api_specs/swagger.yaml
- https://github.com/omgnetwork/elixir-omg/blob/master/apps/omg_watcher_rpc/priv/swagger/security_critical_api_specs/swagger.yaml
- https://github.com/omgnetwork/elixir-omg/blob/master/apps/omg_child_chain_rpc/priv/swagger/operator_api_specs/swagger.yaml

Status value descriptions:
- https://github.com/omgnetwork/elixir-omg/blob/master/apps/omg_watcher_rpc/priv/swagger/security_critical_api_specs/status/schemas.yaml

and etc. etc. for other changes. Basically these are the source files that get compiled into full specs i.e. `info_api_specs.yaml`, `security_critical_api_specs.yaml, etc.

Then followed by compiling to get those 2-3 full specs:
- https://github.com/omgnetwork/elixir-omg/blob/master/apps/omg_child_chain_rpc/priv/swagger/swagger.md
- https://github.com/omgnetwork/elixir-omg/blob/master/apps/omg_watcher_rpc/priv/swagger/swagger.md

**Pull request merged**